src/ansys/geometry/core/connection/product_instance.py

```diff
@@ -348,6 +361,77 @@ def prepare_and_start_backend(
                 GEOMETRY_SERVICE_EXE,
             )
         )
+    # This should be modified to Windows Core Service in the future
+    elif backend_type == BackendType.LINUX_SERVICE:
+        # Define several Ansys Geometry Core Service folders needed
+        root_service_folder = Path(installations[product_version], CORE_GEOMETRY_SERVICE_FOLDER)
+        native_folder = root_service_folder / "Native"
+        cad_integration_folder = root_service_folder / "CADIntegration"
+        schema_folder = root_service_folder / "Schema"
+
+        # Set the environment variables for the Ansys Geometry Core Service launch
+        # ANS_DSCO_REMOTE_IP should be variable "host" directly, but not working...
+        env_copy["ANS_DSCO_REMOTE_IP"] = "127.0.0.1" if host == "localhost" else host
+        env_copy["ANS_DSCO_REMOTE_PORT"] = str(port)
+        env_copy["ANS_DSCO_REMOTE_LOGS_CONFIG"] = "linux"
+        env_copy["P_SCHEMA"] = schema_folder.as_posix()
+        env_copy["ANSYS_CI_INSTALL"] = cad_integration_folder.as_posix()
+        env_copy[f"ANSYSCL{product_version}_DIR"] = (
+            root_service_folder / "licensingclient"
+        ).as_posix()
+
+        if os.name == "nt":
+            # Modify the PATH variable to include the path to the Ansys Geometry Core Service
+            env_copy["PATH"] = (
+                f"{env_copy['PATH']}"
+                + f";{root_service_folder.as_posix()}"
+                + f";{native_folder.as_posix()}"
+                + f";{cad_integration_folder.as_posix()}"
+            )
+
+            # For Windows, we need to use the exe file to launch the Core Geometry Service
+            args.append(
+                Path(
+                    installations[product_version],
+                    CORE_GEOMETRY_SERVICE_FOLDER,
+                    CORE_GEOMETRY_SERVICE_EXE,
+                )
+            )
+        else:
+            # Verify dotnet is installed
+            import shutil
+
+            if not shutil.which("dotnet"):
+                raise RuntimeError(
+                    "Cannot find 'dotnet' command. "
+                    "Please install 'dotnet' to use the Ansys Geometry Core Service."
```

Comment:
Maybe here we could mention that at least 8.0 version is required. So people don't install an older version the first time and have to do it again.

Reply:
That's actually a good idea. I'll do that, thanks @MaxJPRey